### ISSUE TITLE
Link to scripting API docs is dead

### ISSUE THREAD

**Issue by vith** [autokey-py3/autokey-py3#27]
The `scripting API` link in https://github.com/guoci/autokey-py3/blob/42e5a29b07c7ea63db4df8a2153b894fe068ae1e/README.rst#documentation is dead.

Maybe use [GitHub Pages](https://pages.github.com/) to host it?

**Comment by leword** [comment]
:-(   I'm sad.  I really like autokey, I actually pretty much depend on it for sane consistent keybindings in linux..  Not having a reference to the scripting API really hurts, and getting no response on even fixing a link for 4 months doesn't seem to bode well for future of the project...

**Comment by troxor** [comment]
Scripting API copied to https://autokey-py3.github.io/, links in the README and around the codebase have been updated. Don't be sad!

**Comment by leword** [comment]
Thank you!!!   I did eventually find some of what I was looking for by just reading source-code, and it _did_ occur to me that I could try to offer a solution myself, just didn't quite have it me yesterday, so this is very helpful & much appreciated.

**Comment by dlewis7444** [comment]
The new link is also dead now.  But the git project has all the docs in it.  Yay!

I've been using Autokey for a few weeks now for really basic use.  I just wanted a simple key combo to always paste a string in whatever window I happened to be in.  For that, Autokey has been working fine.

Today, I finally started playing with scripting.  And I'm beginning to realize the true power that is Autokey.  I'm in love!
